src/gtk/widgets/recent_info.rs

```diff
@@ -25,12 +25,13 @@ struct_Widget!(RecentInfo);
 
 impl RecentInfo {
     pub fn _ref(&self) -> Option<RecentInfo> {
-        let tmp_pointer = unsafe { ffi::gtk_recent_info_ref(GTK_RECENT_INFO(self.unwrap_widget())) };
+        let tmp_pointer = unsafe {
+            ffi::gtk_recent_info_ref(GTK_RECENT_INFO(self.unwrap_widget()))
+        };
 
-        if tmp_pointer.is_null() {
-            None
-        } else {
-            Some(gtk::FFIWidget::wrap_widget(tmp_pointer as *mut ffi::C_GtkWidget))
+        match tmp_pointer.is_null() {
```

Comment:
I count 7 occurences of 'false =>` in Rust source code. Doesn't seem very idiomatic. The effort could be spent better on eliminating the `if` altogether.